CHANGELOG.md

```diff
@@ -82,6 +82,7 @@
 
 ### Features
 
+* allow fetching other datasets (obf, opff, opf) ([#223](https://github.com/openfoodfacts/openfoodfacts-python/pull/223))
```

Comment:
btw @raphael0202 I think there are some issues with the projet's Changelog, the PR's should only allow "Squash and merge" (and not "Rebase and merge")

Reply:
On the repositories I work on, I do rebase & merge from time to time when I want to keep each atomic commits. On Open Prices, I don't remember, I don't merge commits often ;)

Reply:
release-please works better with "squash & merge"
https://github.com/googleapis/release-please
> We **highly** recommend that you use squash-merges when merging pull requests.

Reply:
for instance, if you look at the CHANGELOG, and search for the PR #274 title "increase csv field_size_limit to accommodate large fields", it links to a commit, not the actual PR...

Reply:
I updated the merge strategy to "Squash & merge" only. I unfortunately used "rebase" for another PR I merged before applying the settings, but it should be the last one ;)